tokens/source/animation/timing.json

```
@@ -0,0 +1,8 @@
+{
+  "animation_timing": {
+    "transition": {
+      "comment": "Default transition timing",
+      "value": "0.2s"
+    }
+  }
+}
```

We'll want to follow the CSS property values for this file (e.g. `animation-duration`), so instead of this `timing.json` file, I would recommend an `animation/index.json` file with this structure:

```suggestion
{
  "animation": {
    "duration": {
      "20": {
        "comment": "Default transition timing",
        "value": "0.2s"
      }
    }
  }
}
```

In the future, we can add other properties like `animation-delay`, etc.

---

Ahh I see. Done.